spark/src/main/kotlin/com/adevinta/spark/components/divider/Divider.kt

```diff
@@ -62,6 +71,13 @@ public enum class LabelVerticalAlignment {
     "Divider is deprecated",
     ReplaceWith("HorizontalDivider()"),
 )
+/**
+ * A deprecated Divider Component.
```

Comment:
We need to add the `@Deprecated` annotation with a `ReplaceWith` that will change it into `HorizontalDivider`

Reply:
It exists     "Divider is deprecated",
    ReplaceWith("HorizontalDivider()"),
)

I will put after the doc tho